src/test/system_tests.cpp

```diff
@@ -49,11 +49,7 @@ BOOST_AUTO_TEST_CASE(run_command)
     }
     {
         // An invalid command is handled by Boost
-#ifdef WIN32
-        const int expected_error{wine_runtime ? 6 : 2};
-#else
         const int expected_error{2};
-#endif
```

Comment:
9a04ca1cd0921c80f4836a07f2a2198fbd43cf05

The error reporting has been changed in Wine 8.0:
- for Wine 6.x and 7.x, it was:
	- message: "CreateProcess failed: No such device or address"
	- code: 6
- Wine 8.0:
	- message: "CreateProcess failed: File not found"
	- code: 2

I'm not sure about the exact commit, though.

I agree with these changes and don't want to introduce more testing code to handle different Wine versions.

However, I'm suggesting to document that Wine 8+ is required to run Windows unit tests.

Reply:
Do you want to split commit this into a separate PR, and add the documentation?

Reply:
I guess this will also means anyone that wants to run the windows unit tests will need to be running Lunar or later, or install Wine themselves.

Reply:
Or skip/ignore this particular test.